Sources/ContainerCommands/Application.swift

```diff
@@ -77,6 +77,12 @@ public struct Application: AsyncParsableCommand {
                     VolumeCommand.self
                 ]
             ),
+            CommandGroup(
```

Comment:
For the `container` CLI we've tried to avoid putting commands not related to container resources directly under `container`. Could you remove this command group, and make it so that the `version` command is a subcommand of `container system`?